test/verify/check-storage-partitions

```diff
@@ -42,7 +42,8 @@ class TestStorage(StorageCase):
 
         dev = m.execute("dd if=/dev/zero of=/disk1 bs=1M count=10 && losetup --show loop12 /disk1").strip()
 
-        b.wait(lambda: b.is_present("#others") and dev in b.text("#others"))
+        b.wait_js_func('function (dev) { return ph_is_present("#others") && ph_text("#others").indexOf(dev) >= 0; }',
+                       dev)
```

Comment:
From @martinpitt:

The function wrapper is unnecessary - `b.wait_js_cond('ph_is_present("#others") && ph_text("#others").indexOf(dev) >= 0') is a bit simpler. But that's just FYI, I'm not vetoing this.

Strictly speaking this is not any more or less "atomic" than the previous variant. However, this has much less overhead as you don't need to go between Python, Chrome, and JS for each iteration, so it's a nice improvement for sure.

Reply:
> The function wrapper is unnecessary 

The function is there so that we can pass in the `dev` value in a clean way, without doing ugly string interpolation.

Reply:
(But I guess we can write `(dev) => ph_is_present ...` now.)

Reply:
> Strictly speaking this is not any more or less "atomic" than the previous variant. 

Oh.  I guess this is mostly not a problem since the DOM usually evolves "monotonically" towards the desired state and once it reaches it, it stays there.

The problem here is that `#others` comes and goes a couple of times until our device shows up in it.  I'll check why that actually happens.